packages/contracts/test/token/erc20/governance-wrapped-erc20.ts

```diff
@@ -486,5 +495,419 @@ describe('GovernanceWrappedERC20', function () {
+            });
+          });
+        });
+      });
+
+      context('`to` has a non-zero balance', async () => {
+        beforeEach(async () => {
+          await expect(token.depositFor(to.address, 100))
+            .to.emit(token, 'DelegateChanged')
+            .withArgs(to.address, addressZero, to.address);
+          toDelegate = to.address;
+          expect(await token.balanceOf(to.address)).to.eq(100);
+        });
+
+        context('`to` delegated to `other`', async () => {
+          beforeEach(async () => {
+            await expect(token.connect(to).delegate(other.address))
+              .to.emit(token, 'DelegateChanged')
+              .withArgs(to.address, to.address, other.address); // this changes the delegate from himself (`to`) to `other`
+            toDelegate = other.address;
+          });
+
+          context(
+            '`to` receives via `depositFor` from `address(0)`',
+            async () => {
+              beforeEach(async () => {
+                await expect(token.depositFor(to.address, 100)).to.not.emit(
+                  token,
+                  'DelegateChanged'
+                );
+              });
+
+              it('`to` has the correct voting power', async () => {
+                expect(await token.getVotes(to.address)).to.equal(0);
+              });
+              it('`to`s delegate has not changed', async () => {
+                expect(await token.delegates(to.address)).to.equal(toDelegate);
+              });
+              it('`to`s delegate has the correct voting power', async () => {
+                expect(await token.getVotes(toDelegate)).to.equal(200);
+              });
+
+              context('`to` transfers to `other`', async () => {
+                beforeEach(async () => {
+                  await expect(
+                    token.connect(to).transfer(other.address, 100)
+                  ).to.not.emit(token, 'DelegateChanged');
+                });
+
+                it('`to` has the correct voting power', async () => {
+                  expect(await token.getVotes(to.address)).to.equal(0);
+                });
+                it('`to`s delegate has not changed', async () => {
+                  expect(await token.delegates(to.address)).to.equal(
+                    toDelegate
+                  );
+                });
+                it('`to`s delegate has the correct voting power', async () => {
+                  expect(await token.getVotes(toDelegate)).to.equal(100);
+                });
+              });
+
+              context('`to` delegates to `other`', async () => {
+                beforeEach(async () => {
+                  await expect(token.connect(to).delegate(other.address))
+                    .to.emit(token, 'DelegateChanged')
+                    .withArgs(to.address, other.address, other.address); // `to` re-delegates to `other` again
+                  //toDelegate = other.address;
```

Comment:
remove unused code

Reply:
Fixed in c0d3b11a35f333f3c0dac00e73c4f7f2edb89620